.dockerignore

```diff
@@ -24,6 +24,7 @@ public/tenants/*/sitemap.xml
 public/assets/
 public/machine_learning/data/
 public/tenants/*/machine_learning/data/
+node_modules/
```

Comment:
Hi @javierm, 

Should we remove this line following the same conclusion we reached [here](https://github.com/consuldemocracy/consuldemocracy/pull/5159#discussion_r1350765100)?